uiowa/uiowa
# Admissions find/replace 2024

Resolves https://github.com/uiowa/uiowa/issues/8316

Leveraging the `DOMXPath` strategy I found in P2LbHelper and elsewhere in the codebase.




# How to test

- `git checkout main`
- `ddev blt ds --site=admissions.uiowa.edu && ddev drush @admissions.local uli admin/find-text`
- Search for `Iowa graduates have a 95 percent job/grad school placement rate` without render markup checked.
- Notice all of the instances come from the `Area Of Study Career` field.
- `git checkout admissions_find_replace_2024`
- `ddev drush @admissions.local updb` - notice no errors. should output the nodes that were changed.
- Search the string again. See there are no results.
- Search for `Iowa graduates have a 96 percent job/grad school placement rate` and see results.
- Check one of the recent AoS node saves. See the revision entry and the diff between versions.
- Go to https://admissions.uiowa.ddev.site/student-profiles and see images for the profiles along with additional person meta info.
